webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -6258,6 +6258,13 @@ const AdminDefinition: AdminDefinitionType = {
                             help_text: defineMessage({id: 'admin.experimental.delayChannelAutocomplete.desc', defaultMessage: 'When true, the autocomplete for channel links (such as ~town-square) will only trigger after typing a tilde followed by a couple letters. When false, the autocomplete will appear as soon as the user types a tilde.'}),
                             isDisabled: it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.EXPERIMENTAL.FEATURES)),
                         },
+                        {
+                            type: 'bool',
+                            key: 'ExperimentalSettings.YoutubeReferrerPolicy',
+                            label: defineMessage({id: 'admin.experimental.youtubeReferrerPolicy.title', defaultMessage: 'YouTube Referrer Policy:'}),
+                            help_text: defineMessage({id: 'admin.experimental.youtubeReferrerPolicy.desc', defaultMessage: 'When true, the referrer policy for embedded YouTube videos will be set to "strict-origin-when-cross-origin". When false, the referrer policy will be set to "no-referrer".'}),
```

Comment:
@cwarnermm Can you check these lines?

I guess we should add the final objective of this, which is that this may solve issues with youtube videos previews, but may have some privacy implications.

You may be able to sync with @esarafianou privacy implications.

Reply:
@larkox - I propose the following System Console help text update, subject to @esarafianou's input and approval:

"When true, the referrer policy for embedded YouTube videos will be set to "strict-origin-when-cross-origin" which balances the need to protect user privacy with some degree of referral data to support web functionalities, like analytics, logging, and third-party integrations. When false, the referrer policy will be set to "no-referrer" which enhances user privacy by not disclosing the source URL, but limits the ability to track user engagement and traffic sources in analytics tools."

Reply:
> @pvev I prefer a subsequents PR. I thought there would be no problem because I brought the parameters used in Slack as it is, but I guess it wasn't?

I just use parameters from Slack. If there are useless parameter and reply it, I'll remove it.

Reply:
I think it should contains the content that it can solves the problem that YouTube video preview shows `Video Unavailable`.

Reply:
@Sn-Kinos - which referrer policy results in the ``Video Unavailable`` being resolved?

Reply:
@cwarnermm `no-referrer` (what we used to have) can result in Youtube marking the videos as unavailable. The `strict-origin-when-cross-origin` one solves the issue, but as you said, it exposes information.

Reply:
@Sn-Kinos - I've updated my [proposed text update](https://github.com/mattermost/mattermost/pull/26980#discussion_r1671176960) above to incorporate your feedback.